.github/workflows/system-tests.yaml

```diff
@@ -0,0 +1,15 @@
+name: System tests
+
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  run-system-tests:
+    if: ${{ github.event.label.name == 'trigger-system-tests' }}
+    uses: precice/tutorials/.github/workflows/run_testsuite_workflow.yml@develop
+    with:
+      suites: release_test
+      build_args: PRECICE_REF:${{ github.event.pull_request.head.sha }}
```

Comment:
It looks like we cannot skip providing all the needed arguments at the moment:

```

Run cd tools/tests
  cd tools/tests
  python systemtests.py --build_args=PRECICE_REF:1a04e2901ef0e1c93f9c51fb6e25af937b8c9612 --suites=release_test --log-level=DEBUG
  cd ../../
  shell: /usr/bin/bash -e {0}
Using log-level: DEBUG
Traceback (most recent call last):
  File "/home/precice/runners_root/actions-runner-org/_work/precice/precice/tools/tests/systemtests.py", line 96, in <module>
    main()
  File "/home/precice/runners_root/actions-runner-org/_work/precice/precice/tools/tests/systemtests.py", line 64, in main
    Systemtest(tutorial, build_args, case, reference_result))
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<string>", line 7, in __init__
  File "/home/precice/runners_root/actions-runner-org/_work/precice/precice/tools/tests/systemtests/Systemtest.py", line 125, in __post_init__
    self.__init_args_to_use()
  File "/home/precice/runners_root/actions-runner-org/_work/precice/precice/tools/tests/systemtests/Systemtest.py", line 150, in __init_args_to_use
    raise Exception(
Exception: PYTHON_BINDINGS_REF is needed to be given via --params to instantiate the systemtest for Flow over heated plate
```

Let's provide them here, and work on the defaults mechanism later: https://github.com/precice/tutorials/issues/552

Reply:
provided explicit arguments